en/scripting/setup.md

```diff
@@ -77,75 +72,159 @@ Before writing code, please read the following documentations to learn more abou
 - [Coding Environment Setup](coding-setup.md)
 - [Operating Environment](basic.md)
 
-## Add a Script to a Scene Node
 
-Adding a script to a scene node is actually adding a script component to that node. Select the scene node to which you wish to add a script in the **Hierarchy** panel, at which point the properties of that node will be displayed in the **Inspector** panel. Adding a script component includes the following two ways:
+After the script file is successfully created, rename the file or modify the class name in the code, the file name and class name will no longer affect each other.
 
-1. Drag and drop the script from **Assets** panel directly into the **Inspector** panel.
+- Take `say-hello` as an example, we renamed it to `hello` in **Asset**.
+Re-select it and check the **Inspector**, the code still displays `class say_hello`, which will not change.
+Re-select the node **Node** where the component was just added on the **Hierarchy**, check the **Inspector**, the component name still displays `say_hello`, and it will not change.
```

Comment:
```suggestion
- Take `say-hello` as an example, we renamed it to `hello` in **Asset**.  
Re-select it and check the **Inspector**, the code still displays `class say_hello`, which will not change.  
Re-select the node **Node** where the component was just added on the **Hierarchy**, check the **Inspector**, the component name still displays `say_hello`, and it will not change.
```